reference-implementation/examples/test_vectors.rs

```diff
@@ -420,8 +567,11 @@ impl TestVectors {
         let header = self.header.iter().map(|tv| tv.verify());
         let aes_ctr_hmac = self.aes_ctr_hmac.iter().map(|tv| tv.verify());
         let sframe = self.sframe.iter().map(|tv| tv.verify());
+        let header256 = self.header.iter().map(|tv| tv.verify());
+        let aes256_ctr_hmac = self.aes256_ctr_hmac.iter().map(|tv| tv.verify());
+        let sframe256 = self.sframe.iter().map(|tv| tv.verify());
```

Comment:
```suggestion
        let aes256_ctr_hmac = self.aes256_ctr_hmac.iter().map(|tv| tv.verify());
```

`header256` and `sframe256` just re-run the header and SFrame test vectors, so they're not needed.